pkg/samples/samples.go

```diff
@@ -32,6 +32,22 @@ var fileEnvTemplate = map[string]func(string) string{
 	".rb":   rbPath,
 }
 
+var languageDisplayNames = map[string]string{
+	"java":   "Java",
+	"node":   "Node",
+	"python": "Python",
+	"php":    "PHP",
+	"ruby":   "Ruby",
+}
+
+var displayNameLanguages = map[string]string{
+	"Java":   "java",
+	"Node":   "node",
+	"Python": "python",
+	"PHP":    "php",
+	"Ruby":   "ruby",
+}
```

Comment:
Hm, I wonder if we should have a routine to just reverse the other map on `init()` or something.

Reply:
(Just because it seems a little undesirable that you could add something to one and forget to add it to the other.)

Reply:
Fixed!